pubspec.yaml

```diff
@@ -1,8 +1,14 @@
 name: fiura_ecosystem
 description: A new Flutter project.
+<<<<<<< HEAD
+
+publish_to:
+
+=======
```

Comment:
need to solve this conflicts first  this cant be here!

Reply:
Fixed.